BiasAndFairnessServers/src/controllers/bias_and_fairness.py

```
@@ -0,0 +1,86 @@
+            data_content = await data.read()
+            data_filename = data.filename
+
+            if not model_content or not data_content:
+                raise ValueError("model or data file is empty")
+            if not model_filename or not data_filename:
+                raise ValueError("model or data file name is empty")
+
+            upload_model_record = upload_model(content=model_content, name=model_filename, db=db)
+
+            if not upload_model_record:
+                raise Exception("failes to upload model file")
```

Pylint (3.3.7)

[warning] Raising too general exception: Exception

(W0719)